apps/site/i18n/locales/en.json

```
@@ -292,7 +292,26 @@
       "codeBox": {
         "systemManagerWarning": "is not a Node.js package manager. Please ensure you already have {packageManager} installed.",
         "communityWarning": "Package managers and their installation scripts are not maintained by the Node.js project.",
-        "communityWarningReport": "If you encounter issues, please reach out to the package manager's maintainers."
+        "communityWarningReport": "If you encounter issues, please reach out to the package manager's maintainers.",
+        "installsNvm": "installs nvm (Node Version Manager)",
+        "downloadAndInstallNodejsRestartTerminal": "download and install Node.js (you may need to restart the terminal)",
+        "verifiesRightNodejsVersion": "verifies the right Node.js version is in the environment",
+        "verifiesRoghtNpmVesrion": "verifies the right NPM version is in the environment",
```

as pointed out by https://github.com/nodejs/nodejs.org/issues/6917
```suggestion
        "verifiesRightNpmVesrion": "verifies the right npm version is in the environment",
```